recipes/Getting_Started/Getting_Started_with_Ollama.ipynb

```diff
@@ -18,18 +18,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Install Ollama\n",
+    "The Ollama server can be run locally on your system or in the notebook itself.\n",
```

Comment:
Both methods run the server locally (on the same host as the notebook). Maybe "from a terminal" or "from the notebook"?